src/codewhisperer/util/closingBracketUtil.ts

```diff
@@ -12,22 +12,54 @@ interface bracketMapType {
     [k: string]: string
 }
 
-const bracketMap: bracketMapType = {
+const quotes = ["'", '"', '`']
+
+const closing: bracketMapType = {
     ')': '(',
     ']': '[',
     '}': '{',
+    '>': '<',
+    "'": "'",
+    '"': '"',
+    '`': '`',
+}
```

Comment:
Looks like this can be derived from the openning map so you don't need to maintain both.